testnet/stacks-node/src/config.rs

```diff
@@ -1455,6 +1472,8 @@ pub struct FeeEstimationConfigFile {
     pub cost_metric: Option<String>,
     pub disabled: Option<bool>,
     pub log_error: Option<bool>,
+    pub fee_rate_fuzzer_fraction: f64,
+    pub fee_rate_window_size: u64,
```

Comment:
I think it may make more sense for these to be `Option<>` fields, then if they're `None`, the `FeeEstimationConfig` struct should use _it's_ default values for these fields.

Reply:
I believe if you do not use the `Option` types here, the config parser will always expect the field to be present in the file, see this playground gist: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=32c5c340dde70bf7b210e13ffc95da8b

Reply:
I switched these to toptions but now the questions is, is the "default" handling correct?